test/sanity/issue6113-mac-create-crash/test.py

```diff
@@ -0,0 +1,34 @@
+import time
+import os
+import platform
+import sys
+
+if platform.system() != 'Darwin':
+    print 'Skipped for non Mac platform'
+    sys.exit(0)
+
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from nw_util import *
+
+chrome_options = Options()
+chrome_options.add_argument("nwapp=" + os.path.dirname(os.path.abspath(__file__)))
+
+driver = webdriver.Chrome(executable_path=os.environ['CHROMEDRIVER'], chrome_options=chrome_options)
+driver.implicitly_wait(2)
+try:
+    wait_window_handles
```

Comment:
@rogerwang Added line 13, 14 and 22, please see here, maybe the page doesn't refresh.

Reply:
does the function call need arguments?

Reply:
Oops, updated it.